ext/calendar/calendar.c

```diff
@@ -219,7 +219,7 @@ enum { CAL_MONTH_GREGORIAN_SHORT, CAL_MONTH_GREGORIAN_LONG,
 };
 
 /* for heb_number_to_chars */
-static char alef_bet[25] = "0����������������������";
+static char alef_bet[25] = "0����������������������";
```

Comment:
Is there an way to revert this?

Reply:
You can use `git checkout -p HEAD~ -- ext/calendar/calendar.c` and only chose that block. Then you amend the commit.